zh-CN/mapping_dev.zh_CN

```diff
@@ -1156,184 +1156,184 @@ dlgSgPreferences.tmi"Low-level Properties"=低级属性
+dlgSgRebaseHeadToCommitConfirm.fur%1=这将把工作树分支 “$1” \(HEAD\)的所有提交应用到所选提交。
+dlgSgRebaseHeadToCommitConfirm.hdl%1=是否要将 “$1” 变基到所选提交？
+dlgSgRebaseHeadToCommitConfirm.tle=将 HEAD 变基到选定的提交
+dlgSgRebaseInteractive.btn"Auto-Squash"=自动压缩
+dlgSgRebaseInteractive.btn"Edit Message"=编辑消息
 dlgSgRebaseInteractive.btn"Edit"=编辑
 dlgSgRebaseInteractive.btn"Move Down"=下移
 dlgSgRebaseInteractive.btn"Move Up"=上移
 dlgSgRebaseInteractive.btn"Rebase"=变基
 dlgSgRebaseInteractive.btn"Remove"=移除
-dlgSgRebaseInteractive.btn"Squash"=Squash
+dlgSgRebaseInteractive.btn"Squash"=压缩
 dlgSgRebaseInteractive.col"IDs"=IDs
 dlgSgRebaseInteractive.col"Message"=消息
 dlgSgRebaseInteractive.hdl=重写历史
-dlgSgRebaseInteractive.inf=You may reorder or squash commits according to your needs.
-dlgSgRebaseInteractive.mni"Neighboring Commits"=Neighboring Commits
+dlgSgRebaseInteractive.inf=您可以根据需要重新排序或取消提交。
+dlgSgRebaseInteractive.mni"Neighboring Commits"=相邻提交
 dlgSgRebaseInteractive.mni"To Bottom Commit"=转到底部提交
 dlgSgRebaseInteractive.mni"To Top Commit"=转到顶部提交
-dlgSgRebaseInteractive.tle=Rebase Interactive
+dlgSgRebaseInteractive.tle=交互式变基
 dlgSgRebaseInteractiveMessage.edt"Commit Message"=提交消息
 dlgSgRebaseInteractiveMessage.hdl=编辑提交消息
 dlgSgRebaseInteractiveMessage.inf=提供新的提交消息.
 dlgSgRebaseInteractiveMessage.tle=编辑消息
 dlgSgRebasingAbortConfirm.btn"Abort Rebase"=中止变基
-dlgSgRebasingAbortConfirm.fur=Your working tree is in 'rebasing' state. You may abort rebasing; if you just want to skip the current patch, use Branch \| Rebase \| Rebase HEAD To instead.\n\nAborting will clean any local modification \(by invoking 'git reset --hard'\)!
+dlgSgRebasingAbortConfirm.fur=您的工作树处于变基状态。你可以放弃变基; 如果您只想跳过当前的补丁，请使用 Branch \| Rebase \| Rebase HEAD 改为。\n\n中止将清除任何本地修改 \(通过调用 “git reset --hard”\)！
 dlgSgRebasingAbortConfirm.hdl=你想要中止变基吗?
 dlgSgRebasingAbortConfirm.tle=变基
 dlgSgRecursive(Stage|Unstage).col"Directory"=目录
-dlgSgRecursive(Stage|Unstage).col"Name"=Name
+dlgSgRecursive(Stage|Unstage).col"Name"=用户名
 dlgSgRecursiveStage.btn"Stage"=暂存
 dlgSgRecursiveStage.col"Directory"=目录
-dlgSgRecursiveStage.col"Name"=Name
-dlgSgRecursiveStage.hdl=Save working tree changes in the Index for later commit
+dlgSgRecursiveStage.col"Name"=用户名
+dlgSgRecursiveStage.hdl=保存索引中的工作树更改以供以后提交
 dlgSgRecursiveStage.inf=将选中的文件暂存到索引
 dlgSgRecursiveStage.tle=暂存
 dlgSgRecursiveUnstage.btn"Unstage"=取消暂存
-dlgSgRecursiveUnstage.hdl=Revert staged changes from the Index to the working tree
-dlgSgRecursiveUnstage.inf=Select the files to unstage from the Index.
-dlgSgRecursiveUnstage.tle=Unstage
-dlgSgRemoteFetchMoreNoMoreBranches.fur=远程版本库中存在的所有分支也都已在本地存在。
+dlgSgRecursiveUnstage.hdl=将暂存更改从索引还原到工作树
+dlgSgRecursiveUnstage.inf=从索引中选择要取消暂存的文件。
+dlgSgRecursiveUnstage.tle=取消暂存
+dlgSgRemoteFetchMoreNoMoreBranches.fur=远程仓库中存在的所有分支也都已在本地存在。
 dlgSgRemoteFetchMoreNoMoreBranches.hdl=没有更多的远程分支来获取。
 dlgSgRemoteFetchMoreNoMoreBranches.tle=获取更多
-dlgSgRemoteProperties.chk"Perform background Poll or Fetch"=Perform background Poll or Fetch
```

Comment:
@mstrap In this, "Poll" is wrong spelling? "Pull" ?

Reply:
"Poll" is correct here: it will just perform a 'git ls-remote' which is more light-weight than 'git fetch'.